pixeltable/exec/data_row_batch.py

```diff
@@ -9,19 +9,22 @@
 
 _logger = logging.getLogger('pixeltable')
 
+
 class DataRowBatch:
     """Set of DataRows, indexed by rowid.
 
     Contains the metadata needed to initialize DataRows.
     """
+
     def __init__(self, tbl: catalog.TableVersion, row_builder: exprs.RowBuilder, len: int = 0):
         self.tbl_id = tbl.id
         self.tbl_version = tbl.version
         self.row_builder = row_builder
         self.img_slot_idxs = [e.slot_idx for e in row_builder.unique_exprs if e.col_type.is_image_type()]
         # non-image media slots
         self.media_slot_idxs = [
-            e.slot_idx for e in row_builder.unique_exprs
+            e.slot_idx
```

Comment:
Is that the auto-formatter? I'm not sure this is a good use of vertical space.

Reply:
I backed out the formatting changes for now, until we can discuss. There are some options that can be configured, which we can look into. Note that this one could be condensed onto a single line (which would be even more compact vertically) and then I'm guessing the formatter would leave it alone.